### PR TITLE
Clipici Constantin-Madalin - Curs 3 - Code Challenge 1

### PR DESCRIPTION
**Pull request by MC19-Dev**

### CODE REVIEW

File: app/src/main/java/com/magdamiu/androidfundamentalsmai2021/MainActivity.java [hunk 1]
```diff
@@ -1,131 +1,55 @@
...
-    private TextView textViewGreetings;
 
-    private WebView webView;
-
-    private Spinner spinnerAndroidVersions;
-    private List<String> androidVersions;
-    private ArrayAdapter<String> spinnerAdapter;
+    private EditText emailLogin;
+    private EditText phoneLogin;
+    CheckBox checkBoxLogin;
```
Comment (magdamiu) [inline app/src/main/java/com/magdamiu/androidfundamentalsmai2021/MainActivity.java]: Please use the private access modifier for all the fields

File: app/src/main/java/com/magdamiu/androidfundamentalsmai2021/MainActivity.java [hunk 2]
```diff
@@ -1,131 +1,55 @@
...
+        buttonLogin = findViewById(R.id.buttonLogin);
+        textLogin = findViewById(R.id.textLogin);
+    }
+
+    public void displaySubmitOnClick(View view) {
+        textLogin.setText("");
+        String email = emailLogin.getText().toString();
+        String phone = phoneLogin.getText().toString();
+        final String successSubmit = "Successfully submitted";
+        if (email.length() > 0 && Patterns.EMAIL_ADDRESS.matcher(email).find()) {
```
Comment (magdamiu) [inline app/src/main/java/com/magdamiu/androidfundamentalsmai2021/MainActivity.java]: To make the code more readable I suggest implementing a function to validate if a string is a valid email address. Same idea for phone number.

**Review by magdamiu — COMMENTED** [review]
checked

File: app/src/main/java/com/magdamiu/androidfundamentalsmai2021/MainActivity.java (hunk 2 above)
Comment (MC19-Dev) [reply app/src/main/java/com/magdamiu/androidfundamentalsmai2021/MainActivity.java]: solved

File: app/src/main/java/com/magdamiu/androidfundamentalsmai2021/MainActivity.java (hunk 1 above)
Comment (MC19-Dev) [reply app/src/main/java/com/magdamiu/androidfundamentalsmai2021/MainActivity.java]: solved

**Pull request closed without merging**